tests/playground.spec.js

```diff
@@ -0,0 +1,69 @@
+import { test, expect } from "@playwright/test";
+const AxeBuilder = require("@axe-core/playwright").default;
+
+const username = "_test-profile-user-1";
+
+test("Playground has title", async ({ page }) => {
+  await page.goto("/playground");
+  await expect(page).toHaveTitle(/Playground/);
+});
+
+test("Navigate to Playground", async ({ page }) => {
+  await page.goto("/");
+  await page.locator("a:visible", { hasText: "Playground" }).click();
+  await expect(page.locator("h1")).toHaveText("Playground");
+});
+
+test("Playground Opens correctly", async ({ page }) => {
+  // 1. Navigate to playground page.
+  await page.goto("playground");
```

Comment:
I think this should be 
```suggestion
  await page.goto("/playground");
```